rpcs3/Emu/Cell/lv2/sys_fs.h

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include "Emu/IdManager.h"
```

Comment:
don't add this here.
You can just move the constructor+member init to the cpp...

Reply:
Done. Thanks for advice.